packages/kit/src/core/build/index.js

```diff
@@ -51,6 +51,9 @@ export async function build(config, { cwd = process.cwd(), runtime = '@sveltejs/
 		}),
 		output_dir,
 		client_entry_file: `${SVELTE_KIT}/build/runtime/internal/start.js`,
+		service_worker_register:
+			config.kit.serviceWorker?.register === undefined ? true : config.kit.serviceWorker.register,
```

Comment:
The CI is failing and it's probably because of `?.`, which doesn't work on older versions of Node